components/WalletsCarousel.js

```diff
@@ -53,7 +53,7 @@ const nStyles = StyleSheet.create({
 const NewWalletPanel = ({ onPress }) => {
   const { colors } = useTheme();
   const { width } = useWindowDimensions();
-  const itemWidth = width * 0.82 > 375 ? 375 : width * 0.82;
+  const itemWidth = width * 0.92 > 375 ? 375 : width * 0.92;
```

Comment:
What is this? 🤔  (same below)

Reply:
the carousel items are smaller to indicate that you can swipe through them and see the next one. As we don't have additional wallets, I thought I will make it bigger until we remove the screen completely, as it looks better (centered)